spec/latest/json-ld/index.html

```diff
@@ -3076,7 +3076,21 @@
     <li>A <a>node</a> having an outgoing edge MUST be an <a>IRI</a> or a
       <a>blank node</a>.</li>
     <li>A <a>graph</a> MUST NOT contain unconnected <a>nodes</a>,
-      i.e., nodes which are not connected by an <a>edge</a> to any other <a>node</a>.</li>
+      i.e., nodes which are not connected by an <a>edge</a> to any other <a>node</a>.
+      <pre class="illegal-example"
+          data-transform="updateExample"
+          title="Illegal Unconnected Node">
+      <!--
+      {
+        "@id": "http://example.org/1"
+      }
+      -->
+      </pre>
+      <div class="note">
+        This effectivly just bans node objects containing only an <code>@id</code>.
```

Comment:
I can see someone (who does not read the rest of the spec, which is common) getting confused by this wording and also thinking this is illegal:

```js
{
  "foo:bar": {
    "@id": "http://example.org/1"
  }
}
```

As that nested node contains only an `@id`. Perhaps a slight tweak:

`This effectively prohibits unnested, empty node objects and unnested node objects that contain only an <code>@id</code>.`